src/Protocol/Protocol_1_8.h

```diff
@@ -104,10 +103,9 @@ class cProtocol_1_8_0:
 	virtual void SendSetTitle                   (const cCompositeChat & a_Title) override;
 	virtual void SendSetRawTitle                (const AString & a_Title) override;
 	virtual void SendSoundParticleEffect        (const EffectID a_EffectID, int a_SrcX, int a_SrcY, int a_SrcZ, int a_Data) override;
-	virtual void SendSpawnFallingBlock          (const cFallingBlock & a_FallingBlock) override;
+	virtual void SendSpawnEntity                (const cEntity & a_Entity) override;
+	virtual void SendSpawnEntity                (const cEntity & a_Entity, const UInt8 a_ObjectType, const Int32 a_ObjectData);
```

Comment:
This should probably be a `protected` member, since it's expected to be called only from the inside of the protocol class.